.github/workflows/tox.yml

```diff
@@ -66,6 +66,14 @@ jobs:
           python3 -m tox
         env:
           TOXENV: ${{ matrix.tox_env }}
+      - name: Upload coverage data
+        uses: codecov/codecov-action@v2
+        with:
+          directory: .tox
+          files: 'coverage-*.xml'
+          name: ${{ matrix.tox_env }}
```

Comment:
I also recommend adding flags — they are useful for determining which jobs/envs covered specific lines.